maestro/financial-connections/Livemode-Data-Finbank.yaml

```diff
@@ -26,7 +26,7 @@ tags:
 # LOG IN WITH TEST FINBANK INSTITUTION
 - extendedWaitUntil:
     visible:
-      text: "Image of FinBank logo"
+      text: "Banking Userid "
```

Comment:
captured using maestro studio (note this is a matcher within a web browser instance)